build/vaultwarden/build.sh

```diff
@@ -24,7 +24,7 @@ DESC="Unofficial Bitwarden compatible server written in Rust, formerly known "
 DESC+="as bitwarden_rs"
 
 DANIGARCIA=$GITHUB/dani-garcia
-WEBVAULTVER=2022.5.2
```

Comment:
Yeah it seems vaultwarden full releases happen less often then importing the upstream patched webvault from bitwarden. Most of the times when a new release of vaultwarden is done it includes the then current version  of the webvault. Let me add a DASHREV guess you just remove that when vaultwarden itself get a newer version. It a bit similar to the COMPONENT_REVSION on oi-userland but there its for the build system as otherwise the package
doesn't get rebuild.

Reply:
b.t.w. does that DASHREV also end up in doc/packages ? Guess not.

Reply:
Thanks for doing that, I think it's better to have a more visible change in the package version for a change like this. Yes, DASHREV just gets removed (or reset to 0) when there's a vaultwarden update.
It doesn't go in doc/packages.md